app/models/Country.scala

```diff
@@ -29,6 +29,8 @@ object Country {
 
   implicit val format: OFormat[Country] = Json.format[Country]
 
+  val unitedKingdom = Country("GB", "United Kingdom")
```

Comment:
Public methods should have type to avoid warnings.

Reply:
Changed.

Seems unnecessary as Scala has type inference.